getting-started/quickstart-guides/php.md

```diff
@@ -42,425 +42,1483 @@ It will prompt a few options. Select defaults by pressing enter.
 composer require imagekit/imagekit
 ```
 
-#### Initialize SDK
+## Quick Examples
+
+#### Create an ImageKit Instance
 
 ```php
 // Import autoloader from vendor
 // If not using PSR-4 is not configured in composer.json file for your project
 require_once __DIR__ . '/vendor/autoload.php';
 
-use ImageKit\ImageKit;
-
+use ImageKit\ImageKit;  
+  
 // For demonstration purposes, the documentation would use https://ik.imagekit.io/demo as urlEndpoint
 $imageKit = new ImageKit(
-    "{publicKey}",
-    "{publicKey}",
-    "{urlEndpoint}" 
+    "publicKey",
+    "privateKey",
+    "urlEndpoint"
 );
-
 ```
+* `publicKey` and `privateKey` parameters are required as these would be used for all ImageKit API, server-side upload, and generating token for client-side file upload. You can get these parameters from the developer section in your ImageKit dashboard - [https://imagekit.io/dashboard#developers](https://imagekit.io/dashboard#developers).
+* `urlEndpoint` is also a required parameter. You can get the value of URL-endpoint from your ImageKit dashboard - [https://imagekit.io/dashboard#url-endpoints](https://imagekit.io/dashboard#url-endpoints).&#x20;
 
-* `urlEndpoint` is the required parameter. You can get the value of URL-endpoint from your ImageKit dashboard - [https://imagekit.io/dashboard#url-endpoints](https://imagekit.io/dashboard#url-endpoints).&#x20;
-* `publicKey` and `privateKey` parameters are also required as these would be used for all ImageKit API, server-side upload, and generating token for client-side file upload. You can get these parameters from the developer section in your ImageKit dashboard - [https://imagekit.io/dashboard#developers](https://imagekit.io/dashboard#developers).
-
-## Generating url for rendering images
-
-#### URL for image with relative path&#x20;
-
+#### URL Generation
 ```php
-$imageURL = $imageKit->url(['path' => '/default-image.jpg']);
-
-echo('Url : ' . $imageURL); 
-// https://ik.imagekit.io/demo/default-image.jpg
+// For URL Generation
+$imageURL = $imageKit->url(
+    [
+        'path' => '/default-image.jpg',
+    ]
+);
+echo $imageURL;
 ```
 
-![](../../.gitbook/assets/default-image.jpeg)
-
-#### URL for image with relative path and custom URL-endpoint
-
+#### File Upload
 ```php
-$imageURL = $imageKit->url([
-      'urlEndpoint' => 'https://ik.imagekit.io/test',
-      'path' => '/default-image.jpg',
+// For File Upload
+$uploadFile = $imageKit->upload([
+    'file' => 'file-url',
+    'fileName' => 'new-file'
 ]);
+```
 
-echo('Url : ' . $imageURL); 
-// https://ik.imagekit.io/test/default-image.jpg
+#### Response Structure
+```json
+{
+    "error": {},
+    "result": {},
```

Comment:
What is the point of showing an empty `{}` in the quick example? It should be populated with legitimate vales and link of doc should be added - https://docs.imagekit.io/api-reference/upload-file-api/server-side-file-upload#response-code-and-structure-json